docs/notebook_tutorial/ocr.ipynb

```diff
@@ -80,7 +89,8 @@
     "        # Read in Image data and store in DataPack\n",
     "        # add Image payload into DataPack.payloads\n",
     "        image_data = Image.open(requests.get(data_source, stream=True).raw)\n",
-    "        pack.set_image(image_data, payload_idx)\n",
+    "        width, height = image_data.size\n",
+    "        ip.set_cache(image_data, [height, width])\n",
```

Comment:
Do we need to explicitly set the shape here? `set_cache` can automatically infer the image shape when the input data is `ndarray`.